### ISSUE TITLE
Remote config not found

### ISSUE THREAD

**Issue by ReneFroger** [eshion/vim-sftp-sync#3]
I have a index.php file, stored in the folder Test.
I put my SFTP settings in vimrc and in a config file in the Test folder of the editted file, called 'configs'. 
The settings are: 

let g:vim_sftp_configs = {
\      'sample_server_1' : {
\       'upload_on_save'   : 1,
\       'download_on_open' : 0,
\       'confirm_downloads': 1,
\       'confirm_uploads'  : 0,
\       'local_base_path'  : '/Users/Rene Froger/Desktop/Test/',
\       'remote_base_path' : '/httpdocs/',
\       'sftp_command' : 'ftp',
\       'user' : 'xxxxxxxx',
\       'pass' : 'xxxxxxxx',
\       'host' : 'xxxxx'
\   }
}

I use as command: :call SftpUpload()
Then I get the message "Remote config not found". What went wrong here?

**Comment by vodich** [comment]
Hi did you solve this?

**Comment by ReneFroger** [comment]
Switched to Emacs. I suggest you do the same, the Vim inside Spacemacs is even better.
In Emacs you can everything there as you do in Vim with Evil. Everything. Vim modal editing, Vim commands like `:%s/foo/bar/g` `:e foobar.txt, :cd :vsplit` or another ex commands, and even import your Vimrc, if you need a marco. Emacs even interprets Vimscripts, if you want to write a marco for it.  But now you have a full blown development environment, complete keyboard driven operation system without any fuss or GUI. There are packages in Emacs that have no Vim equivalence. Some of them are listed on http://emacs.zeef.com

You can read [this comment written](https://www.reddit.com/r/emacs/comments/2rmp6b/emacs_as_my_leader_1_year_later/cnhtcpf) by a Vim convert. [This one is](http://nathantypanski.com/blog/2014-07-02-switching-to-emacs.html) even better.

There is a preconfigured Emacs package with Vim included. [You can test it for yourself](https://github.com/syl20bnr/spacemacs). But you can also [set up Emacs with Evil yourself.](http://nathantypanski.com/blog/2014-08-03-a-vim-like-emacs-config.html). 

Interactive REPL stuffs? Vim still [has to go](https://www.reddit.com/r/vim/comments/383s10/looking_for_suggestions_about_vim_and_repl/crspgix) a long way. Even Neovim is not able to do [this](http://i.imgur.com/LnRwOwd.gifv) or [this](https://github.com/millejoh/emacs-ipython-notebook/wiki/Screenshots). [And much more, see a awesome list here.](https://www.reddit.com/r/emacs/comments/32ekmg/dear_emacs_users_of_reddit_how_do_you_use_emacs/cqahmqo). 

You can also check this video: [how I use Vim with Emacs](http://www.youtube.com/watch?v=JWD1Fpdd4Pc&t=20m20s). 

## Summary
To put it simply. Emacs is capable of more. There are many things that Emacs can do that Vim simply cannot. This is because Emacs can display images, process tasks asynchronously in the background, and be truly extended, By this I mean that all of Emacs’ core functionality is exposed through elisp functions, which can be called and even overridden using a sort of mixin strategy called “advice.” Given this, there are few limitations to what you can do to customize it. Vim provides a scripting language that can manipulate most of Vim’s behavior, but it is limited primarily to modifying data; you cannot change any fundamental behaviors like how Vim draws line numbers. And it's all free and opensource.

**Comment by vodich** [comment]
Ah don't know if I am ready to do that just now. Ill probably give it a go once I find more spare time to get into it. Thanks for all those infos on emacs!

**Comment by ReneFroger** [comment]
Try Spacemacs to figure out already if it's something for you.

**Comment by zhaogaolong** [comment]
I use windows gvim 
myconfig:
`

let g:vim_sftp_configs = {
\	'centos7_dev' : {
\       'upload_on_save'   : 1,
\    	'download_on_open' : 1,
\    	'confirm_downloads': 0,
\    	'confirm_uploads'  : 0,
\		'local_base_path'  : 'H:\TestPorject\',
\		'remote_base_path' : '/root/tmp/sync/',
\		'sftp_command' : 'sftp',
\       'complete_prompt_regexp' : '100\%',
\		'user' : 'xxx',
\		'pass' : 'xxx',
\		'host' : 'xxx.xxx.xxx.xxx'
\	}
\}

nnoremap <C-U> <ESC>:call SftpUpload()<CR>
nnoremap <C-D> <ESC>:call SftpDownload()<CR
`

I use as command: :call SftpUpload()
Then I get the message "Remote config not found". What went wrong here?

**Comment by hmohdarafat** [comment]
Where do we put the config file? I get the error  "Remote config not found" even if I place the config in the _vimrc file. I use Windows.
